packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -674,6 +677,25 @@ exports.extendNodeType = ({ type, store, reporter }) => {
     })
   }
 
+  const getDominantColor = async ({ image, options }) => {
+    const { rgbToHex } = require(`gatsby-plugin-sharp`)
```

Comment:
I'd wrap this in a try/catch block. Sharp is only a peerdep, and is only needed for this, so it would be best to fail gracefully if it's missing.

Reply:
Oh wait, we're using it for blurred too, yeah?